std/random.d

```diff
@@ -1020,7 +1020,7 @@ alias Mt19937_64 = MersenneTwisterEngine!(ulong, 64, 312, 156, 31,
     // Check .save works
     static foreach (Type; std.meta.AliasSeq!(Mt19937, Mt19937_64))
     {{
-        auto gen1 = Type(unpredictableSeed);
+        auto gen1 = Type(123_456_789); gen1.popFront();
```

Comment:
Better move the `popFront` into a newline, people don't like this short style:

> One statement per line.

(from https://dlang.org/dstyle.html#whitespace)